client/css/style.css

```diff
@@ -1475,6 +1475,20 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	display: inline-block;
 }
 
+.whois {
+	display: grid;
+	grid-template-columns: max-content auto;
+}
+
+.whois dt {
+	min-width: 150px;
```

Comment:
Is there a way to make the second column start at "<width of longest text in first column> + margin" instead of a fixed width?
Thinking of those with different font family or size for example.